### PR TITLE
Lowercase relative paths of recommendations

### PR DESCRIPTION
**Pull request by chezou**
When file name of an article has upper case, Hugo generates URL as lowercased path. It should respect their behavior.

Note that I'm not sure other generator behaviors, so please let me know if we need to have option for it.

### CODE REVIEW

**Comment by takuti** [comment]
The argument generally sounds reasonable, but I'd propose to make it configurable at `StaticSiteGeneratorHandler` level.

- I guess we both are testing on Hugo, and the behavior is actually [configurable on Hugo](https://gohugo.io/getting-started/configuration/#disablepathtolower).
- There is no definite answer whether URL must be case-sensitive; eventually, it's a mix of developer's preference and web server's setting cf. [Jekyll issue](https://github.com/jekyll/jekyll/issues/2443), [stackoverflow discussion](https://stackoverflow.com/questions/48388057/how-to-make-github-pages-url-case-insensitive/48389490).

**Comment by chezou** [comment]
Okay, let me it configurable.

**Comment by chezou** [comment]
Fixed in 5c40cb3

File: prelims/handler.py [hunk 1]
```diff
@@ -14,10 +14,12 @@ def __init__(self, path_dir, ignore_files=None, encoding='utf-8'):
         self.processors = []
         self.encoding = encoding
         self.ignore_files = [] if ignore_files is None else ignore_files
+        self.lower_path = lower_path
 
     def register_processor(self, processor):
         """Add a front matter processor to the queue.
         """
+        processor.lower_path = self.lower_path
```
Comment (takuti) [inline prelims/handler.py]: `processor` can be anything other than `Recommender`, and it's not always true the processor has an attribute `lower_path`. I feel you don't need to update `StaticSitePostsHandler` for the time being; as long as `Recommender` is instantiated with its own `lower_path` attribute, it should do the job.

Apologies my suggestion "configureable at StaticSitePostsHandler level" was misleading; I was thinking something more generic and widely-applicable implementation (in the context of #6), but you don't need to worry about it.

Comment (chezou) [reply prelims/handler.py]: Ah, okay 011e793

**Comment by chezou** [comment]
@takuti No rush, but let me know if this change will be released on PyPI.

**Comment by takuti** [comment]
@chezou ✅ https://pypi.org/project/prelims/0.0.6/

**Pull request merged**